network/p2p/tracer/gossipSubMeshTracer.go

```diff
@@ -167,26 +168,36 @@ func (t *GossipSubMeshTracer) logPeers() {
 	defer t.topicMeshMu.RUnlock()
 	for topic := range t.topicMeshMap {
 		shouldWarn := false // whether we should warn about the mesh state
-		lg := t.logger.With().Dur("heartbeat_interval", t.loggerInterval).Str("topic", topic).Logger()
+
+		roleDict := zerolog.Dict()
+		peerIdDict := zerolog.Dict()
+		flowIdDict := zerolog.Dict()
```

Comment:
instead of splitting the info up, could we just group the 3 values by peer? I think that would make them easier to reason with. If the concern is the overhead from duplicate labels, we could use `pid`/`fid`/`rl`, or even `p`/`f`/`r`

Reply:
Applied in https://github.com/onflow/flow-go/pull/4035/commits/f4de1cab573c7b030ed3ead84656aec360ba84a2

Now the logs look like this:
```
{
    "level": "warn",
    "component": "gossip_sub_topology_tracer",
    "heartbeat_interval": 1000,
    "topic": "push-blocks/a7ba34736ba1b25b750624d88f3a3f9d0690ddfc4de5655a21ee2a803ccc2362",
    "topic_mesh": {
        "0": "pid=16Uiu2HAkvguZPfmbg7MRFT7oS6bW1BXMtAQA1395Qk5orVCjZm7Z, flow_id=61974709abcf01d4c9ef279b49c4560e776af3a1d5888756b99b0f607a732fd7, role=consensus",
        "1": "pid=16Uiu2HAmCvLg84FncK7vziwfaDhWk4rjuXEtJChHCKToUDoF7RHM, flow_id=unknown, role=unknown",
        "2": "pid=16Uiu2HAmJbMMTMcXo4PpC4gzH2EVY8c52cM3SvDQvntVGGGSBtoD, flow_id=5496b70e7cd9ba3859307a6cfe308c5cbea1f16edd9be2d2a4a67cf05c589eb4, role=consensus"
    },
    "suspicious": true,
    "message": "unknown peers in topic mesh peers of local node since last heartbeat"
}
```